typescript/examples/model-context-protocol-smart-wallet-server/package.json

```diff
@@ -19,6 +19,7 @@
     "@coinbase/agentkit": "workspace:*",
     "@coinbase/agentkit-model-context-protocol": "workspace:*",
     "@modelcontextprotocol/sdk": "^1.6.1",
+    "viem": "^2.23.14",
```

Comment:
i see that we are updating to ^2.24.x in other package.json, let's do that here as well